src/test/java/com/andmcadams/wikisync/WikiSyncLogPlugin.java

```diff
@@ -0,0 +1,31 @@
+package com.andmcadams.wikisync;
+
+import ch.qos.logback.classic.Level;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+import ch.qos.logback.classic.Logger;
+import net.runelite.client.ui.overlay.OverlayManager;
+import org.slf4j.LoggerFactory;
+
+@Singleton
+@PluginDescriptor(
+	name = "[Debug] WikiSync Logging"
+)
+public class WikiSyncLogPlugin extends Plugin
+{
+
+	@Override
+	protected void startUp()
+	{
+		((Logger) LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME)).setLevel(Level.WARN);
+		((Logger) LoggerFactory.getLogger("com.andmcadams.wikisync")).setLevel(Level.DEBUG);
+	}
+
+	@Override
+	protected void shutDown()
+	{
+		((Logger) LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME)).setLevel(Level.DEBUG);
+	}
+}
```

Comment:
It looks like this is just turning on debug for wikisync only. Is that the case? If so this is a much better way of doing it than we have done in the past lol

Reply:
Yep! @llemonduck helped me with this